rust/lance-encoding/src/encoder.rs

```diff
@@ -127,6 +132,12 @@ pub trait ArrayEncoder: std::fmt::Debug + Send + Sync {
 /// A task to create a page of data
 pub type EncodeTask = BoxFuture<'static, Result<EncodedPage>>;
 
+/// A buffer of encoded metadata to be placed in the column metadata
+pub struct EncodedMetadataBuffer {
```

Comment:
Is there a reason to have a new type here instead of `EncodedBuffer` (which currently has the same fields)?

Reply:
Good catch.  I'll switch to `EncodedBuffer`.  That helps reinforce the fact that there isn't much special between the two.

Reply:
Switched.